README.md

```diff
@@ -123,6 +123,19 @@ cp /usr/i686-w64-mingw32/bin/lib{gcc_s_dw2-1,ssp-0,stdc++-6,winpthread-1}.dll /m
 wine /my/path/to/Winamp/winamp.exe
 ```
 
+## Building & Running on Windows
+
+Don't. But if you must there are to ways to build
+
+1. open the folder with Visual Studio
+2. use the generate_project_files.bat script, which produces a 32-bit only Visual Studio project using CMake.
+
+If you don't want to do either, but use CMake in some other way here are some caveats to be aware of:
```

Comment:
I think this whole section is unneeded. _If_ you are using CMake in some unholy way, you probably know CMake, and are aware of GUI problems and caches etc.